java/src/main/java11/module-info.java

```diff
@@ -0,0 +1,79 @@
+module io.cucumber.java {
+    exports io.cucumber.java;
+    exports io.cucumber.java.af;
+    exports io.cucumber.java.am;
+    exports io.cucumber.java.an;
+    exports io.cucumber.java.ar;
+    exports io.cucumber.java.ast;
+    exports io.cucumber.java.az;
+    exports io.cucumber.java.bg;
+    exports io.cucumber.java.bm;
+    exports io.cucumber.java.bs;
+    exports io.cucumber.java.ca;
+    exports io.cucumber.java.cs;
+    exports io.cucumber.java.cy_gb;
+    exports io.cucumber.java.da;
+    exports io.cucumber.java.de;
+    exports io.cucumber.java.el;
+    exports io.cucumber.java.en;
+    exports io.cucumber.java.en_au;
+    exports io.cucumber.java.en_lol;
+    exports io.cucumber.java.en_old;
+    exports io.cucumber.java.en_pirate;
+    exports io.cucumber.java.en_scouse;
+    exports io.cucumber.java.eo;
+    exports io.cucumber.java.es;
+    exports io.cucumber.java.et;
+    exports io.cucumber.java.fa;
+    exports io.cucumber.java.fi;
+    exports io.cucumber.java.fr;
+    exports io.cucumber.java.ga;
+    exports io.cucumber.java.gj;
+    exports io.cucumber.java.gl;
+    exports io.cucumber.java.he;
+    exports io.cucumber.java.hi;
+    exports io.cucumber.java.hr;
+    exports io.cucumber.java.ht;
+    exports io.cucumber.java.hu;
+    exports io.cucumber.java.id;
+    exports io.cucumber.java.is;
+    exports io.cucumber.java.it;
+    exports io.cucumber.java.ja;
+    exports io.cucumber.java.jv;
+    exports io.cucumber.java.ka;
+    exports io.cucumber.java.kn;
+    exports io.cucumber.java.ko;
+    exports io.cucumber.java.lt;
+    exports io.cucumber.java.lu;
+    exports io.cucumber.java.lv;
+    exports io.cucumber.java.mk_cyrl;
+    exports io.cucumber.java.mk_latn;
+    exports io.cucumber.java.mn;
+    exports io.cucumber.java.nl;
+    exports io.cucumber.java.no;
+    exports io.cucumber.java.pa;
+    exports io.cucumber.java.pl;
+    exports io.cucumber.java.pt;
+    exports io.cucumber.java.ro;
+    exports io.cucumber.java.ru;
+    exports io.cucumber.java.sk;
+    exports io.cucumber.java.sl;
+    exports io.cucumber.java.sr_cyrl;
+    exports io.cucumber.java.sr_latn;
+    exports io.cucumber.java.sv;
+    exports io.cucumber.java.ta;
+    exports io.cucumber.java.th;
+    exports io.cucumber.java.tl;
+    exports io.cucumber.java.tlh;
+    exports io.cucumber.java.tr;
+    exports io.cucumber.java.tt;
+    exports io.cucumber.java.uk;
+    exports io.cucumber.java.ur;
+    exports io.cucumber.java.uz;
+    exports io.cucumber.java.vi;
+    exports io.cucumber.java.zh_cn;
+    exports io.cucumber.java.zh_tw;
+    requires io.cucumber.core;
```

Comment:
Because of `io.cucumber.core.api.Scenario` shouldn't this be `requires transitive`? 

Not sure if I'd like that. Perhaps we should wrap `Scenario` in a to be created `io.cucumber.java.Scenario`?

Reply:
at the moment i'm just looking at `requires`, before looking cleaning up more and sorting out `requires transitive`